sql/updates/world/4.3.4/2021_10_10_00_world.sql

```diff
@@ -0,0 +1,3847 @@
+UPDATE `creature` SET `position_x`= 1070.115, `position_y`= 1506.748, `position_z`= 31.02911 WHERE `guid` = 322042;
+
+DELETE FROM `creature_addon` WHERE `guid` = 322043;
+INSERT INTO `creature_addon` (`guid`, `waypointPathId`, `cyclicSplinePathId`, `mount`, `bytes1`, `bytes2`, `emote`, `aiAnimKit`, `movementAnimKit`, `meleeAnimKit`, `visibilityDistanceType`, `auras`) VALUES 
+(322043, 3220430, 0, 0, 0, 1, 0, 0, 0, 0, 0, '18950');
+ 
+DELETE FROM `waypoint_data` WHERE `id` = 3220430;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `velocity`, `delay`, `smoothTransition`, `move_type`, `action`, `action_chance`, `wpguid`) VALUES 
+(3220430, 0, 1070.11500, 1506.748, 31.02911, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 1, 1092.72000, 1511.68, 29.87494, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 2, 1109.26000, 1513.38, 32.26623, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 3, 1122.57000, 1517.24, 34.48486, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 4, 1125.36000, 1537.25, 31.45801, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 5, 1124.48000, 1559.41, 30.35254, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 6, 1123.99000, 1587.8, 29.22581, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 7, 1120.64000, 1614.32, 29.03091, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 8, 1120.64000, 1614.32, 29.03091, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 9, 1092.70000, 1618.21, 27.43043, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 10, 1064.64000, 1619.24, 28.03432, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 11, 1022.61000, 1615.55, 23.80789, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 12, 1010.27000, 1608.76, 24.62271, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 13, 1010.56000, 1592.86, 25.89364, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 14, 1005.36000, 1575.49, 28.67892, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 15, 1003.64000, 1555.16, 28.84166, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 16, 1004.17000, 1535.15, 32.03062, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 17, 1010.66000, 1527.75, 33.21204, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 18, 1037.60100, 1528.172, 31.5291, NULL, 2.509363, 0, 1, 1, 0, 0, 0),
+(3220430, 19, 1048.53300, 1509.726, 32.8853, NULL, 2.509363, 0, 1, 1, 0, 0, 0);
+
+DELETE FROM `creature_formations` WHERE `MemberGUID` IN (322043, 322042);
+INSERT INTO `creature_formations` (`LeaderGUID`, `MemberGUID`, `FollowDistance`, `FollowAngle`, `GroupAI`, `InversionPoint1`, `InversionPoint2`) VALUES
+(322043, 322043, 0, 0, 515, -1, -1),
+(322043, 322042, 3.5, 270, 515, -1, -1);
+
+ --
+ -- The Warchief Cometh Quest
+ --
+ 
+ -- Garrosh Hellscream
+UPDATE `broadcast_text` SET `Text` = 'Remember, Sylvanas, eventually we all have to stand before our maker and face judgment. Your day may come sooner than others...' WHERE `ID` = 44721;
```

Comment:
need to investigate

Reply:
Apparently, this misspelling error was fixed in a patch older than 5.4.7, so we're leaving it as it is, but then it got the error again in 9.1.5.